backends-velox/src/main/scala/org/apache/spark/memory/GlobalOffHeapMemory.scala

```diff
@@ -0,0 +1,57 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.spark.memory
+
+import org.apache.gluten.exception.GlutenException
+
+import org.apache.spark.{SparkEnv, TaskContext}
+import org.apache.spark.storage.BlockId
+
+import java.lang.reflect.Field
+import java.util.UUID
+
+object GlobalOffHeapMemory {
+  private val FIELD_MEMORY_MANAGER: Field = {
+    val f = classOf[TaskMemoryManager].getDeclaredField("memoryManager")
+    f.setAccessible(true)
+    f
+  }
+
+  def acquire(numBytes: Long): Boolean = {
+    memoryManager().acquireStorageMemory(
+      BlockId(s"test_${UUID.randomUUID()}"),
+      numBytes,
+      MemoryMode.OFF_HEAP)
```

Comment:
it looks like we are using OFF_HEAP only, not sure if this will impact on the fallback case

Reply:
This API is supposed to be called only from Gluten's operators. So we don't need to consider the fallback case yet.